Projects/AccountSDKIOSWebExample/AnotherSharedKeychainApp/ContentView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  ContentView.swift
+//  AnotherSharedKeychainApp
+//
+//  Created by Daniel Echegaray on 2021-11-21.
+//
+
+import SwiftUI
+import AccountSDKIOSWeb
+
+struct ContentView: View {
+    static let config = ClientConfiguration(environment: .pre,
+                                            clientId:"App TWO ID",
+                                            redirectURI: URL(string: "com.sdk-example.pre.602504e1b41fa31789a95aa7:/login")!)
+                                            
+    static let client: Client =  Client(configuration: ContentView.config)
+
+    var body: some View {
+        Text("Hello, world!")
+            .padding()
+        
+        VStack(spacing: 50) {
+            VStack(spacing: 20) {
+                Button(action: fetchFromSharedKeychain, label: { Text("fetch from Shared keychain") } )
+                Button(action: storeToSharedKeychain, label: { Text("Store To Shared keychain") } )
+            }
+        }
+    }
+    
+    func storeToSharedKeychain() {
+        let myAccessGroup = "GS8T83EM2X.com.testing.SharingExample"
+        let manager = SimplifiedLoginManager(accessGroup: myAccessGroup, client: ContentView.client)
+        manager.storeInSharedKeychain(clientId: "AppOne", aStringValue: "THIS IS APP ONE STORING :D") { result in
```

Comment:
ClientId is included in ClientConfiguration, which is included in Client which is included in SimplifiedLoginManager. Maybe instead of passing clientId, we can retrieve it? 
Also i think is a good idea to add property clientId (which underneath reads value from associated ClientConfiguration) directly to the Client.  Should help a lot and it sounds pretty natural to have id of an object as a property.